components/Spotlight.js

```diff
@@ -0,0 +1,18 @@
+import Image from "next/image";
+
+export default function Spotlight({ image, artist }) {
+  return (
+    <>
```

Comment:
You don't need the Fragments, since you have only one root Element (`article`) inside it. But you can adjust this in the next feature, so I can give you a green check right now :)